services/src/main/java/org/broadleafcommerce/payment/service/gateway/DefaultPayPalCheckoutTransactionService.java

```diff
@@ -473,19 +441,19 @@ private final PayPalResource salePayment(PaymentRequest paymentRequest)
         return response.getSale();
     }
 
-    private final Payer generateSalePayer(PaymentRequest paymentRequest) {
+    protected Payer generateSalePayer(PaymentRequest paymentRequest) {
         if (isBillingAgreementRequest(paymentRequest)) {
             return generateBillingAgreementPayer(paymentRequest);
         }
 
         return null;
     }
 
-    private final List<Transaction> generateSaleTransactions(PaymentRequest paymentRequest) {
+    protected List<Transactions> generateSaleTransactions(PaymentRequest paymentRequest) {
         return generateTransactions(paymentRequest);
     }
```

Comment:
Is the purpose of this to be an extension point?

Reply:
Yeah - if you intend for your sale (auth & capture) transactions to be different from your other trasactions